facial_recognition/face_reco.py

```diff
@@ -0,0 +1,88 @@
+"""
+@author: Adel Abu Hashim
+"""
+
+import face_recognition as face
+import data
+
+# import known_faces from data file.
+known_faces = data.known_faces
+
+def init ():
+    '''
+    add test data
+    '''
+    global known_faces
+    # load test images
+    image_of_adel = face.load_image_file("adel.jpg")
+    image_of_luis = face.load_image_file("luis.jpg")
+    image_of_josh = face.load_image_file("josh.jpg")
+    # extract face encodings for each face in numpy array
+    face_encoding_of_adel = face.face_encodings(image_of_adel)[0]
+    face_encoding_of_luis = face.face_encodings(image_of_luis)[0]
+    face_encoding_of_josh = face.face_encodings(image_of_josh)[0]
+    # store each person name and his face encodings-converted from numpy array to list- in dict.
```

Comment:
aren't these already stored in `dict.txt`?

Reply:
I replaced dict.txt with data.py but I forgot to delete first one; I pushed a new commit in which I have deleted it and also I've done some clean for data.

Reply:
thank you, but now `data.py` is missing!

Reply:
![image](https://user-images.githubusercontent.com/35013845/77653506-73eea600-6f78-11ea-9322-2687dc2f2cf8.png)

Reply:
yes, it's empty, it should have some data, and then load this data in `init` /?

Reply:
I got your mean, added!